### PR TITLE
BHBC-1930, 1931, 1932

### PR DESCRIPTION
**Pull request by AlfredRosenthal**
# Overview

## Links to Jira tickets

[BHBC-1930](https://quartech.atlassian.net/browse/BHBC-1930?atlOrigin=eyJpIjoiNzdmOTY0NDljNWIzNGJmZmE3ZGVkYWVhOWFlMGQ2NzciLCJwIjoiaiJ9)

[BHBC-1931](https://quartech.atlassian.net/browse/BHBC-1931?atlOrigin=eyJpIjoiYWMwYWUzMTA0MTQ2NGQ2MjkxYzU3YjRiMmQyNDIwZGMiLCJwIjoiaiJ9)

[BHBC-1932](https://quartech.atlassian.net/browse/BHBC-1932)

## Description of relevant changes

- Added SRB, non SRB and Aerial Transect Elk templates to resource page
- Added SRB, non SRB and Aerial Trasect validation migrations

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

File: database/src/migrations/template_methodology_species_validations/elk_non_srb_survey.ts [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+              'Survey or Telemetry Search',
+              'Photos',
+              'Observation Comments'
+            ]
+          }
+        }
+      ]
+    },
+    {
+      name: 'MArked Animals',
```
Comment (curtisupshall) [inline database/src/migrations/template_methodology_species_validations/elk_non_srb_survey.ts]: Typo.

File: database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts [hunk 2]
```diff
@@ -251,20 +252,31 @@ const Habitat = [
...
   { name: 'Caves' },
   { name: 'Clearcut' },
+  { name: 'Clearcut Herbaceous' },
   { name: 'Cliff' },
   { name: 'Cliff - open' },
   { name: 'Cliff in Forest' },
   { name: 'Coniferous' },
+  { name: 'Cultivated Field' },
+  { name: 'Cutblock - Free to Grow' },
+  { name: 'Cutblock - Matrue' },
```
Comment (curtisupshall) [inline database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts]: Should this read `'Cutblock - Mature'`?

**Review by curtisupshall — CHANGES_REQUESTED** [review]

File: database/src/migrations/template_methodology_species_validations/elk_non_srb_survey.ts (hunk 1 above)
Comment (AlfredRosenthal) [reply database/src/migrations/template_methodology_species_validations/elk_non_srb_survey.ts]: good catch, fixed.

File: database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts (hunk 2 above)
Comment (AlfredRosenthal) [reply database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts]: fixed

**Review by KjartanE — APPROVED** [review]
🥇 Looks great

**Review by curtisupshall — APPROVED** [review]

**Pull request merged**